### DISCUSSION TITLE
Support for Definition of Done (DoD) section

### DISCUSSION THREAD

**Discussion by jfalatowicz** [ankitpokhrel/jira-cli#478] [Ideas] [answered]
I can currently edit/view the "Description" section of any issue.
It would be useful to add support for the Definition of Done (DoD) section of issues.

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @jf13, I don't think this is a standard section available in all installations. If this is a custom field, using `--custom` flag to set this field should work.

**Reply by bdunn313** [reply]
I can confirm this is a non-standard, custom field. This is the right answer!
